airbyte-integrations/connectors/destination-kafka/src/main/resources/spec.json

```diff
@@ -0,0 +1,223 @@
+        "type": "integer",
+        "default": 16384,
+        "order": 13
+      },
+      "linger_ms": {
+        "title": "Linger ms",
+        "description": "The producer groups together any records that arrive in between request transmissions into a single batched request.",
+        "type": "integer",
+        "default": 0,
+        "order": 14
+      },
+      "max_in_flight_requests_per_connection": {
+        "title": "Max in flight requests per connection",
+        "description": "The maximum number of unacknowledged requests the client will send on a single connection before blocking.",
+        "type": "integer",
+        "default": 5,
+        "order": 15
+      },
+      "client_dns_lookup": {
+        "title": "Client DNS lookup",
+        "description": "Controls how the client uses DNS lookups.",
+        "type": "string",
+        "default": "use_all_dns_ips",
+        "enum": [
+          "default",
+          "use_all_dns_ips",
+          "resolve_canonical_bootstrap_servers_only",
+          "use_all_dns_ips"
+        ],
+        "order": 16
+      },
+      "buffer_memory": {
+        "title": "Buffer memory",
+        "description": "The total bytes of memory the producer can use to buffer records waiting to be sent to the server.",
+        "type": "integer",
+        "default": 33554432,
+        "order": 17
+      },
+      "max_request_size": {
+        "title": "Max request size",
+        "description": "The maximum size of a request in bytes.",
+        "type": "integer",
+        "default": 1048576,
+        "order": 18
+      },
+      "retries": {
+        "title": "Retries",
+        "description": "Setting a value greater than zero will cause the client to resend any record whose send fails with a potentially transient error.",
+        "type": "integer",
+        "default": 2147483647,
+        "order": 19
+      },
+      "socket_connection_setup_timeout_ms": {
+        "title": "Socket connection setup timeout",
+        "description": "The amount of time the client will wait for the socket connection to be established.",
+        "type": "integer",
+        "default": 10000,
+        "order": 20
+      },
+      "socket_connection_setup_timeout_max_ms": {
+        "title": "Socket connection setup max timeout",
+        "description": "The maximum amount of time the client will wait for the socket connection to be established. The connection setup timeout will increase exponentially for each consecutive connection failure up to this maximum.",
+        "type": "integer",
+        "default": 30000,
+        "order": 21
+      },
+      "delivery_timeout_ms": {
+        "title": "Delivery timeout",
+        "description": "An upper bound on the time to report success or failure after a call to 'send()' returns.",
+        "type": "integer",
+        "default": 120000,
+        "order": 22
+      },
+      "send_buffer_bytes": {
```

Comment:
can we relieve the user of needing to think about this by choosing a sane default?

Reply:
Yes, I set a default value